src/app/CMakeLists.txt

```diff
@@ -145,10 +150,11 @@ if (STACKTRACE)
 endif()
 
 if (GUI)
-    set_target_properties(qbt_app PROPERTIES OUTPUT_NAME qbittorrent)
     target_link_libraries(qbt_app PRIVATE qbt_gui)
-else()
-    set_target_properties(qbt_app PROPERTIES OUTPUT_NAME qbittorrent-nox)
+    if ((CMAKE_SYSTEM_NAME STREQUAL "Windows") OR (CMAKE_SYSTEM_NAME STREQUAL "Darwin"))
+        # NOTE: Migrate to qt_import_plugins() once support of Qt < 5.154 is dropped.
```

Comment:
What about adding a branch instead of a comment? `if (Qt5_VERSION VERSION_LESS 5.14) ... else() ... endif()`

Reply:
Well, done.